base/applications/mspaint/lang/ro-RO.rc

```diff
@@ -105,18 +105,18 @@ ID_POPUPMENU MENU
 BEGIN
     POPUP "Popup #0"
     BEGIN
-        MENUITEM "C&ut\tCtrl+X", IDM_EDITCUT
-        MENUITEM "Copy\tCtrl+C", IDM_EDITCOPY
-        MENUITEM "&Paste\tCtrl+V", IDM_EDITPASTE
-        MENUITEM "De&lete selection\tDel", IDM_EDITDELETESELECTION
-        MENUITEM "Select &All\tCtrl+A", IDM_EDITSELECTALL
+        MENUITEM "&Decupează\tCtrl+X", IDM_EDITCUT
+        MENUITEM "Copiază\tCtrl+C", IDM_EDITCOPY
```

Comment:
Since all the other menu options have access key assigned, you may assign something to this one too. ("C" is free for the taking.)